### ISSUE TITLE
Example to sign excel file

### ISSUE THREAD

**Issue by samuraitruong** [PeculiarVentures/xadesjs#114]
Is there any example for XaDes-T signature for the excel file?

**Comment by rmhrisk** [comment]
No but a PR for one would be appreciated.

This library would handle most of the lifting, would also need zip library.

it’s relatively straightforward format.

**Comment by samuraitruong** [comment]
@rmhrisk are we going to unzip the file, sign all the XML then zip it again?

I will give it a try , hopefully, I can have PR for it

**Comment by rmhrisk** [comment]
Yes, the format, as I recall is a zip file with XML manifests referencing XML other XML files. The manifest itself has signed not the other files.

This project shows how to work with the format https://github.com/unidoc/unioffice, the signing will be straightforward with that scaffolding in place.

**Comment by samuraitruong** [comment]
@rmhrisk  gave it the first try last night, I can generate all the manifest but the signature does not show up :) will try more. I think all the refs and XML need to be updated properly according to open XML format specs
